docs/root/version_history/current.rst

```diff
@@ -5,6 +5,8 @@ Changes
 -------
 
 * access loggers: added GRPC_STATUS operator on logging format.
+* access loggers: added a virtual method serializeAsString to FilterState for obtaining unstructured serialization results and
```

Comment:
Rather than talk about implementation details, maybe express this as what's been adding from a user facing perspective and link with `:ref:` to your new docs above.